enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
@bdero Ah ok, yep, I misunderstood.  Sounds like the `EnterpriseCustomerContactInfo` 1-1 relationship with `EnterpriseCustomer`, with just the single `contact_email` field is fine at this point.